src/main/java/com/develhope/imsdevelhope/app/persistence/entities/Items.java

```diff
@@ -0,0 +1,80 @@
+package com.develhope.imsdevelhope.app.persistence.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "ITEMS")
+public class Items {
+    public Integer itemId;
+    public String itemName;
+    public boolean itemAvailable;
+    public Integer itemStock;
+    @Id
+    private Long id;
+
+
+    public Items(Integer itemId, String itemName, boolean itemAvailable, Integer itemStock) {
```

Comment:
it would be better if you use a no args constructor.

Reply:
There's a noargs  constructor. Or what do you mean by noargs constructor.